tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -501,7 +501,11 @@ def _test_bed(self):
             try:
                 while 1:
                     topo_line = f_topo.readline().strip()
+		    if re.match(r'#.*',topo_line):
```

Comment:
It seems this will match any line containing `#`, is that what we want?

Reply:
Yes, any line start with # is comment which we don't need this